docs/root/version_history/current.rst

```diff
@@ -1,5 +1,5 @@
-1.15.0 (Pending)
-================
+1.15.0 (July 6, 2020)
```

Comment:
Need to `git mv current.rst v1.15.0.rst`, and then add an entry in `version_history.rst`

Reply:
governance.md says to do that after cutting the release.  let me dig through logs and see what we did last time

Reply:
Oh, we haven't done one since the new docs set up.
I'm cool either way. Want me to move the file and update the instructions?

Reply:
Sorry, my mistake. Follow the governance.